ovs/lib/albacontroller.py

```diff
@@ -285,6 +287,21 @@ def remove_cluster(alba_backend_guid):
         if len(albabackend.asds) > 0:
             raise RuntimeError('A backend with claimed OSDs cannot be removed')
 
+        # openvstorage nodes
+        for abm_service in albabackend.abm_services:
+            test_ip = abm_service.service.storagerouter.ip
+            try:
+                SSHClient(test_ip, username='root')
```

Comment:
SSHClient caches its clients
So if this client would have been created eariler by another process, a client will be available in the cache and this client might have been able to connect. While at this point in time, the connectivity might be lost ==> execute a simple call over the connection to actually verify connectivity